Kernel/Assertions.h

```diff
@@ -15,7 +15,7 @@
 #    define VERIFY_NOT_REACHED() VERIFY(false)
 #else
 #    define VERIFY(expr)
-#    define VERIFY_NOT_REACHED() _abort()
+#    define VERIFY_NOT_REACHED() __builtin_unreachable()
```

Comment:
Actually, the whole point of `VERIFY` is that it *should* be compiled into release builds.
If you want something to be compiled out, it should be using `ASSERT` (which you will have to add first :)

That said, the fact that we *do* compile `VERIFY` out in `!DEBUG` builds is a bug!

Reply:
@gunnarbeutner maybe leave out this commit if everything works without it?

Reply:
Aye, that's the plan. I just haven't gotten around to this yet with all the LibJS bytecode stuff :fire: .